### ISSUE TITLE
(macOS) Notice About Remote Content In Mail.app No Longer Displaying

### ISSUE THREAD

**Issue by bit-blit** [mullvad/mullvadvpn-app#4538]
With the latest version of Mullvad (2023.3), I noticed that in Mail.app it no longer informs me about remote content - I get no notice at the top of the email that all external content has been routed through my VPN. Not sure if this is Mullvad as there have been two OS updates in the past month. Bottom line is that it doesn't alert me anymore. It's not the end of the world, but it was nice to see. Anyone else notice this?

macOS Ventura 13.3.1
Mullvad 2023.3

**Comment by bit-blit** [comment]
Still not getting this message in Mail.app. I don't believe this is an OS issue. Can anyone confirm this is the case with macOS 13.3.1 and Mullvad 2023.3? Really would like to know as that little notice is VERY important to me.

Anyone? Bueller?

Thanks!

**Comment by pinkisemils** [comment]
I don't believe this has anything to do with the Mullvad client. Could you elaborate why you'd believe this is the case.

**Comment by bit-blit** [comment]
Hello, and thank you for contacting me.

The problem didn’t start until I upgraded to 2023.3. 2023.2 works correctly with no problems.

Any help would be greatly appreciated.

Many thanks.

Kind Regards,
Gigo

**Comment by bit-blit** [comment]
> @bit-blit do you mean this banner in Main app? This seems to be present and I use 2023.4 beta on macOS 13.2.1 (tethering via iPhone).
> 
> <img alt="Screenshot 2023-05-05 at 14 36 14" width="507" src="https://user-images.githubusercontent.com/704044/236459078-0cf1b81b-00a1-4a04-b373-765bdeca1bfd.png">

Yes, that is correct. I no longer get this dialog and am not sure why. Seems to have started right after I went to V2023.3

I assume all remote content is being loaded through my VPN even without getting the message because I am connected to it, but I no longer get that dialog which was nice to see. Now knowing that you do get it makes me think there is a conflict of some sort. What are your Mail.app settings for "Privacy" - Do you have this activated? There may be some kind of conflict there on my end - not sure. Any help would be greatly appreciated.

**Comment by pronebird** [comment]
@bit-blit I have "Hide IP address" enabled in Privacy in Mail.app. Maybe something changed in the recent macOS that caused this banner to disappear?

**Comment by bit-blit** [comment]
> @bit-blit I only have "Hide IP address" enabled in Privacy in Mail.app. Maybe something changed in the recent macOS that caused this banner to disappear?

I changed Mail.app Privacy settings to just "Hide IP Address" as you do, and the problem still persists (I thought the two may be having a conflict of some kind). Later today, when I get time, I am going to revert to an older version of Mullvad temporarily just to see if the dialog problem resolves. If it does, then I will know it is something with Mullvad. Of course, I will then install the current version again as I need to be using the latest version for stability/security reasons. Although that dialog is technically unessential, it was nice to see, and I would like to get it working properly again. Many thanks for your help.

**Comment by Rawa** [comment]
Closing as we didnt hear back from the user, feel free to reply if this is still present with your findings.
